patches/minecraft/net/minecraft/client/resources/TextureAtlasHolder.java.patch

```diff
@@ -0,0 +1,14 @@
+--- a/net/minecraft/client/resources/TextureAtlasHolder.java
++++ b/net/minecraft/client/resources/TextureAtlasHolder.java
+@@ -45,4 +_,11 @@
+    public void close() {
+       this.f_118884_.m_118329_();
+    }
++
++   /**
++    * {@return the set of sprites in the atlas.}
++    */
++   public java.util.Set<ResourceLocation> getTextureLocations() {
++      return f_118884_.getTextureLocations();
++   }
```

Comment:
Why not return the texture atlas instead, and let the user call the other newly-added helper method to access the sprites?